test/jdk/javax/swing/AbstractButton/5049549/bug5049549.java

```diff
@@ -101,6 +113,14 @@ private static void runTest() {
 
         KButton button;
 
+        Icon DE = generateImage("DE");
```

Comment:
If you prefer local variables, I'd move the declaration of `KButton` below the icons.